core/src/main/java/tc/oc/pgm/destroyable/DestroyableFactory.java

```diff
@@ -10,15 +11,37 @@
 import tc.oc.pgm.modes.Mode;
 import tc.oc.pgm.teams.TeamFactory;
 import tc.oc.pgm.util.material.MaterialMatcher;
+import tc.oc.pgm.util.xml.InvalidXMLException;
 
 @FeatureInfo(name = "destroyable")
 public class DestroyableFactory extends ProximityGoalDefinition {
+  public static enum SparksType {
+    SPARKS_ALL("true"),
+    SPARKS_NONE("false"),
+    SPARKS_NEAR("near");
```

Comment:
```suggestion
    NONE("false"),
    NEAR("near"),
    ALL("true");
```
I think it makes more sense in order, from least to most (or if you prefer from most to least), but not with jumps. Also since this is a SparkType there's no need to prefix it with SPARKS all the time